nginx/timings.py

```diff
@@ -1,6 +1,7 @@
 import re
 import time
 from datetime import datetime
+from dateutil import parser
```

Comment:
hmm this may not work. datadog doesn't run with a virtualenv so my guess is this will throw an import error. we could globally install, or figure out how to run it in a python env